.github/workflows/test.yml

```diff
@@ -11,28 +11,17 @@ jobs:
       - name: Check out Git repository
         uses: actions/checkout@v3
 
-      - name: Use Node.js
-        uses: actions/setup-node@v3
+      - name: Setup environment
+        uses: prezly/setup-github-actions@v1
         with:
-          node-version-file: '.nvmrc'
-
-      - name: Cache node modules
-        uses: actions/cache@v3
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: "**/node_modules"
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
+          pnpm: 9.15.4
+          node: 20
 
       - name: Install Node.js dependencies
-        run: npm ci
+        run: pnpm install
```

Comment:
Should we use `--frozen-lockfile` here?

Reply:
https://github.com/prezly/theme-kit-js/pull/844/commits/4b9e4875d7cdf564173fbb51e3ad967c64a34de0